trunner/target/armv7m4.py

```diff
@@ -48,7 +48,10 @@ def __init__(self, harness: Callable[[TestResult], TestResult]):
         self.harness = harness
 
```

Comment:
```suggestion
        # Set of additional config parameters used in Openocd to flash up stm32l4a6
        self.extra_args = ["-c", "reset_config srst_only srst_nogate connect_assert_srst", "-c", "init;reset"]
```

Reply:
whaaat?  This is different `self`, please provide the extra args correctly (via a constructor to different class as it's now)

Reply:
Ooh right, I confused `STM32L4x6OpenocdGdbServerHarness` with `STM32L4x6OpenocdGdbServer`, I will remove wrong suggestions.